static/app/components/hovercard.tsx

```diff
@@ -75,18 +74,19 @@ function Hovercard({
   const hovercardContent = isOpen && (
     <PositionWrapper zIndex={theme.zIndex.hovercard} {...overlayProps}>
       <StyledHovercard
+        showArrow
+        arrowProps={{
+          ...arrowProps,
+          size: 20,
+          background: tipColor,
+          border: tipBorderColor,
+        }}
```

Comment:
`showArrow` makes it explicit whether we will display an arrow or not. An alternative to this is to automatically display an arrow if `arrowProps` is defined – this is a less explicit rendering condition but it would also mean less code.

Reply:
I am 👍 to `arrowProps` enabling the arrow I think. UNLESS you can do `showArrow` and it just puts an arrow at whatever the placement is by default. Then I think it's fine